xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -346,10 +440,18 @@
         <Parameter Name="unmanagedDllName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="unmanagedDllName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="unmanagedDllName">Name of the native unmanaged DLL to be loaded without its extension</param>
+        <summary>Loads an unmanaged DLL by simple name.</summary>
```

Comment:
Again - "simple name" is misleading. I would simple state "Loads an unmanaged DLL specified by the <paramref unmanagedDllName/>"